core/src/edu/cornell/gdiac/physics/Timer.java

```diff
@@ -0,0 +1,62 @@
+package edu.cornell.gdiac.bubblebound;
+
+public class Timer {
+
+    private int maxTime;
+
+    private int currentTime;
+
+    boolean hasFinished;
+
+    boolean pause;
+
+    public Timer(int newMax){
+        maxTime = newMax;
+        currentTime = maxTime;
+        pause = false;
+        hasFinished = false;
+    }
+
+    public Timer(int newMax, boolean isPause){
+        maxTime = newMax;
+        currentTime = maxTime;
+        pause = isPause;
+        hasFinished = false;
+    }
+
+    public int getTime(){
+        return currentTime;
+    }
+
+    public void setTime(int value){
+        currentTime = value;
+    }
+
+    public int getMaxTime(){
+        return maxTime;
+    }
+
+    public void setMaxTime(int value){
+        maxTime = value;
+    }
+
+    public void update(){
+        if(!(hasFinished || pause)) {
+            currentTime--;
```

Comment:
I think the idea was to have the timer be based in seconds and not game ticks

but that depends on whether we want to make the fps set to a default value or not